server/szurubooru/tests/func/test_posts.py

```diff
@@ -388,6 +387,7 @@ def test_update_post_content_to_existing_content(
             'post_height': 300,
         },
         'secret': 'test',
+        'allow_broken_uploads': False,
```

Comment:
I'd separate this logic around preventing broken uploads into a separate pull request

Reply:
I think it's necessary for us to properly define the error handling characteristics before we merge in this PR. If really needed, we could break this off and merge it first. But if we are going to merge it anyway I would rather save on the effort. 🙃

Reply:
Yeah makes sense, I'd also like to avoid having to review it twice